### ISSUE TITLE
why not for standard eco without screen

### ISSUE THREAD

**Issue by gabbiano69** [jatin69/vitamin-guide-amazon-alexa-skill#1]
it will be fantastic make avaible also for eco dot with audio?
can you modify it? because i try it and this is the result
Invalid Directive
vitamin guide
Request Identifier: null

The device does not support Alexa.Presentation.APL directives

Feedback vocale

**Comment by jatin69** [comment]
Yes, I didn't realize it fails on echo dot. I'll fix it in a few days. btw this skill is my entry for alexa multi modal challenge. check it out [here](https://devpost.com/software/vitamin-guide).

Thanks for the review. Appreciate it.

**Comment by gabbiano69** [comment]
any news???

**Comment by jatin69** [comment]
@gabbiano69 The skill has been fixed and is under review.

**Comment by gabbiano69** [comment]
great, may i know wich files have you changed and what exactly do you add for code? if is possible? anyway thanks for yours answear

**Comment by jatin69** [comment]
[This function](https://github.com/jatin69/vitamin-guide-amazon-alexa-skill/commit/38b9487d2488e624ec37fae9f8f9645c4297d2e7#r32379452) is the main addition. Simply search for its occurrences to know all the changes.

**Comment by gabbiano69** [comment]
great! may i suggest a standard card for device audio only? when the device is just audio device , when is open the skill send standard card.
if yiu make this the skill is perfect...

**Comment by jatin69** [comment]
But why? The audio device can't really utilize the standard card. What's the benefit of sending?

**Comment by gabbiano69** [comment]
non for device but for app, companion alexa app on the smartphone, the standard card can help and guide the users or just to send some message for example yours  website

**Comment by gabbiano69** [comment]
any news?

**Comment by gabbiano69** [comment]
?

**Comment by gabbiano69** [comment]
hi jatin can you help me?
i know that you are busy but for you is a work of two minutes,

i have a an alexa skill  (
skill-sample-nodejs-audio-player/*multiple-streams*/

and i tryed to add it apl language

i did welcomeView.json from the support of alexa developper console (design)
and i add the render in lanchrequest in index.js

but this not work

if you see this 2 file (attached) you can see and adjust it for me?
pleaseeeee
i rename index in json to send by mail but is index.js

i'm waiting your answear it's important for me


i have a problem with a skill
*Cordiali saluti, Best regard,*
*Francesco Greco *






Il giorno gio 21 feb 2019 alle ore 17:52 Jatin <notifications@github.com>
ha scritto:

**Comment by jatin69** [comment]
Hey @gabbiano69 I did not receive any email yet. If you attached files to any github issued email, I am not sure it'll reach me. 
Also, i am afraid i cannot take one on one queries right now, or look at files. If you'd describe the problem verbally, i'll try to do my best.
Regards

**Comment by gabbiano69** [comment]
this is the code i add apl in launchrequest, but not work when i test it in
developer console

maybe i can't use apl e standard card togheter?

/* eslint-disable  func-names */
/* eslint-disable  no-console */
/* eslint-disable  no-restricted-syntax */
/* eslint-disable  consistent-return */

const alexa = require('ask-sdk');
const constants = require('./constants');
const welcomeView = require("./welcomeView.json");



const card = {



title: 'Pillole di Coaching',


text: 'di: Massimo Binelli.\n \n www.massimobinelli.it\n \n \
www.atletavincente.com\n \ninfo@massimobinelli.it',

                cardcontent: '
https://www.artigianiinfiera.it/wp-content/uploads/oroscopo/pillole-di-coaching/immagini/logo.png
',



image: {


largeImageUrl: '
https://www.artigianiinfiera.it/wp-content/uploads/oroscopo/pillole-di-coaching/immagini/logo.png
',


smallImageUrl: '
https://www.artigianiinfiera.it/wp-content/uploads/oroscopo/pillole-di-coaching/immagini/logo.png
'





  }

};


/* INTENT HANDLERS */

const LaunchRequestHandler = {
  canHandle(handlerInput) {
    return handlerInput.requestEnvelope.request.type === 'LaunchRequest';
  },
  async handle(handlerInput) {
    const playbackInfo = await getPlaybackInfo(handlerInput);
    let message;
    let reprompt;

    if (!playbackInfo.hasPreviousPlaybackSession) {
      message = 'Benvenuti in pillole di coaching a cura di Massimo
Binelli. Puoi chiedere, inizia, per far partire lultimo episodio del
podcast.';
      reprompt = 'Puoi dire:, inizia, per iniziare.';
    } else {
      playbackInfo.inPlaybackSession = false;
      message = `Stavi ascoltando
${constants.audioData[playbackInfo.playOrder[playbackInfo.index]].title}.Vorresti
riprendere da dove avevi interrotto?`;
      reprompt = 'Puoi dire si per riprendere da dove avevi lasciato o no
per ricominciare dallultimo episodio';
    }




return handlerInput.responseBuilder
      .speak(message)
      .reprompt(reprompt)
      .withStandardCard(card.title, card.text, card.cardcontent,
card.smallImageUrl, card.largeImageUrl)
      .addDirective({
                    type: 'Alexa.Presentation.APL.RenderDocument',
                    version: '1.0',
                    document: welcomeView.document,
                    datasources: welcomeView.dataSources
})

.getResponse();
  },
};


thanks?



Il giorno mar 2 lug 2019 alle ore 14:07 Jatin <notifications@github.com> ha
scritto:

**Comment by gabbiano69** [comment]
?
